TLM/TLM/UI/Localization/Translation.cs

```diff
@@ -24,6 +24,10 @@ public class Translation {
         private const string TUTORIAL_HEAD_KEY_PREFIX = TUTORIAL_KEY_PREFIX + "HEAD_";
         public const string TUTORIAL_BODY_KEY_PREFIX = TUTORIAL_KEY_PREFIX + "BODY_";
 
+        public const string GUIDE_KEY_PREFIX = "TMPE_GUIDE_";
+        private const string GUIDE_HEAD_KEY_PREFIX = GUIDE_KEY_PREFIX + "HEAD_";
```

Comment:
Why is this one `private` and the others `public`?

Reply:
I just copy pasted from *TUTORIAL*.